lam4doc/entrypoints/ui/views.py

```diff
@@ -51,7 +51,9 @@ def download_lam_report():
         else:
             try:
                 with tempfile.TemporaryDirectory() as temp_folder:
-                    file_name = f'LAM-report.{form.report_extension.data}'
+                    report_extension = form.report_extension.data if form.report_extension.data == HTML_REPORT_TYPE \
+                        else ZIP_REPORT_TYPE
+                    file_name = f'LAM-report.{report_extension}'
```

Comment:
how many report types we can then have? 3?
HTML, pdf, zip ?

Reply:
there are 2 types of formats that the service provides:
- HTML
- ZIP (the zip contains the PDF files)